### PR TITLE
logback: sync with 3.X

### PR DESCRIPTION
**Pull request by wdauchy**

### CODE REVIEW

File: templates/default/logback.xml.erb [hunk 1]
```diff
@@ -43,8 +50,39 @@
     </encoder>
   </appender>
 
+  <% if node.cassandra.logback.debug.enable -%>
```
Comment (michaelklishin) [inline templates/default/logback.xml.erb]: This attribute doesn't seem to be documented?

File: templates/default/logback.xml.erb [hunk 2]
```diff
@@ -69,5 +114,6 @@
     <% end -%>
   </root>
 
+  <logger name="org.apache.cassandra" level="DEBUG"/>
```
Comment (michaelklishin) [inline templates/default/logback.xml.erb]: Do we really need default logger configuration to log at `DEBUG` level?

File: templates/default/logback.xml.erb (hunk 1 above)
Comment (wdauchy) [inline templates/default/logback.xml.erb]: done

Comment (michaelklishin) [inline templates/default/logback.xml.erb]: Apparently the "dot access" feature will be gone in Chef 13: https://travis-ci.org/michaelklishin/cassandra-chef-cookbook/builds/188491258. Let's switch to "bracket access". Also, I'm not 100% sure if the CI failure is due to these changes but it seems to be that way.

Comment (wdauchy) [inline templates/default/logback.xml.erb]: If I manage to pass tests without it, I propose to switch it in another commit

**Pull request merged**